src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue

```diff
@@ -93,10 +93,41 @@
 				</div>
 				<div class="row">
 					<div class="columns small-12 large-expand">
-						Lend button
+						<action-button
+							class="expandable-loan-card-action-button"
+							:loan-id="loan.id"
+							:loan="loan"
+							:items-in-basket="itemsInBasket"
+							:is-lent-to="loan.userProperties.lentTo"
+							:is-funded="isFunded"
+							:is-selected-by-another="isSelectedByAnother"
+							:is-simple-lend-button="false"
+
+							@click.native="trackInteraction({
+								interactionType: 'addToBasket',
+								interactionElement: 'Lend25'
+							})"
+
+							@add-to-basket="$emit('add-to-basket', $event)"
+						/>
 					</div>
-					<div class="matching-text columns small-12 large-4">
-						2x matching
+					<div class="columns small-12 large-4 show-for-large matching-text-wrap">
+						<matching-text
+							:matching-text="loan.matchingText"
+							:is-funded="isFunded"
+							:is-selected-by-another="isSelectedByAnother"
+							:wrap="true"
+						/>
+					</div>
+				</div>
+				<div class="row">
```

Comment:
We can actually delete everything from line 123-131 this row as per the comment on line 114